src/prank_parallel.cpp

```diff
@@ -37,33 +39,44 @@ int main(int argc, char** argv){
 
 
 
-    omp_set_num_threads(num_threads);
-
     //PHASE 1 : Read graph from input file
-    //SP: Should we do parallel file read as well ?
+    cout<<"Loading graph data..."<<endl;
```

Comment:
Can prints be shoved inside a debug define? It will make output parsing to extract the time easier.

Reply:
I can do this as needed if you'd like.